og.og_permissions.yml

```diff
@@ -2,10 +2,11 @@ update group:
   title: 'Edit group'
   description: 'Edit the group. Note: This permission controls only node entity type groups.'
   'default role':
-    - OG_ADMINISTRATOR_ROLE
```

Comment:
yeah, it's a good idea, because also if I remove the custom define we have in the unit test, it will fail due to its use in `\Drupal\og\OgPermissionHandler::buildPermissionsYaml`

Any pointer to those 'special' strings?